src/js/components/SelectMultiple/SelectMultipleContainer.js

```diff
@@ -0,0 +1,595 @@
+                  }}
+                />
+              </Keyboard>
+            </Box>
+          )}
+          <Box flex={false}>{help}</Box>
+          <OptionsContainer
+            role="listbox"
+            tabIndex="0"
+            ref={optionsRef}
+            aria-multiselectable
+            onMouseMove={() => setKeyboardNavigation(false)}
+            aria-activedescendant={optionsRef?.current?.children[activeIndex]}
+          >
+            {options.length > 0 ? (
+              <InfiniteScroll
+                items={options}
+                step={theme.select.step}
+                onMore={onMore}
+                replace={replace}
+                show={activeIndex !== -1 ? activeIndex : undefined}
+              >
+                {(option, index, optionRef) => {
+                  const optionDisabled = checkDisabled(
+                    index,
+                    disabled,
+                    disabledKey,
+                    options,
+                    valueKey || labelKey,
+                  );
+                  const optionSelected = value.includes(
+                    valueKey && valueKey.reduce
+                      ? applyKey(option, valueKey)
+                      : option,
+                  );
+                  const optionActive = activeIndex === index;
+                  const optionLabel = getOptionLabel(
+                    index,
+                    options,
+                    labelKey || valueKey,
+                  );
+
+                  // Determine whether the label is done as a child or
+                  // as an option Button kind property.
+                  let child;
+                  let textComponent = false;
+                  if (children) {
+                    child = children(option, index, options, {
+                      active: optionActive,
+                      disabled: optionDisabled,
+                      selected: optionSelected,
+                    });
+                    if (
+                      typeof child === 'string' ||
+                      (child.props &&
+                        child.props.children &&
+                        typeof child.props.children === 'string')
+                    )
+                      textComponent = true;
+                  } else {
+                    child = (
+                      <CheckBox
+                        label={
+                          <Box alignSelf="center" width="100%" align="start">
+                            {optionLabel}
+                          </Box>
+                        }
+                        pad="xsmall"
+                        tabIndex="-1"
+                        checked={optionSelected}
+                        disabled={optionDisabled}
+                      />
+                    );
+                  }
+
+                  if (!children && search) {
+                    if (
+                      typeof optionLabel === typeof '' &&
```

Comment:
Why not `typeof optionLabel === 'string'`?

Reply:
Changed to `typeof optionLabel === 'string'` in latest commit